platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/attributes/AbstractConfigurationAttributesResolveIntegrationTest.groovy

```diff
@@ -622,48 +622,6 @@ All of them match the consumer attributes:
           - Provides flavor 'free' but the consumer didn't ask for it""")
     }
 
-    def "does not select default configuration when no configurations with attributes and default configuration is not consumable"() {
-        given:
-        file('settings.gradle') << "include 'a', 'b'"
-
-        file("a/build.gradle") << """
-            $typeDefs
-
-            configurations {
-                _compileFreeDebug.attributes { $freeDebug }
-            }
-            dependencies {
-                _compileFreeDebug project(':b')
-            }
-            task checkDebug(dependsOn: configurations._compileFreeDebug) {
-                doLast {
-                    assert configurations._compileFreeDebug.collect { it.name } == []
-                }
-            }
-        """
-
-        file("b/build.gradle") << """
-            $typeDefs
-
-            apply plugin: 'base'
-            configurations {
-                foo
-                bar
-                'default' {
-                    canBeConsumed = false
-                }
-            }
-        """
-
-        when:
-        expectConsumableChanging(':b:default', false)
```

Comment:
🤔 Looks like the `expect*Changing` methods in `ConfigurationUsageChangingFixture` are no longer used after this PR, so those can be removed